src/System.Data.OleDb/src/Resources/Strings.resx

```diff
@@ -0,0 +1,1020 @@
+  </data>
+  <data name="OleDbCommandBuilder_DataAdapter" xml:space="preserve">
+    <value>The DataAdapter for which to automatically generate OleDbCommands</value>
+  </data>
+  <data name="OleDbParameter_OleDbType" xml:space="preserve">
+    <value>The parameter native type.</value>
+  </data>
+  <data name="PlatformNotSupported_OleDb" xml:space="preserve">
+    <value>System.Data.OleDb is not supported on this platform.</value>
+  </data>
+  <data name="PlatformNotSupported_GetIDispatchForObject" xml:space="preserve">
+    <value>Marshal.GetIDispatchForObject API not available on this plaTform</value>
+  </data>
+  <data name="ADP_CalledTwice" xml:space="preserve">
+    <value>The method '{0}' cannot be called more than once for the same execution.</value>
+  </data>
+  <data name="ADP_IncorrectAsyncResult" xml:space="preserve">
+    <value>Incorrect async result.</value>
+  </data>
+  <data name="ADP_SingleValuedProperty" xml:space="preserve">
+    <value>The only acceptable value for the property '{0}' is '{1}'.</value>
+  </data>
+  <data name="ADP_DoubleValuedProperty" xml:space="preserve">
+    <value>The acceptable values for the property '{0}' are '{1}' or '{2}'.</value>
+  </data>
+  <data name="ADP_InvalidPrefixSuffix" xml:space="preserve">
+    <value>Specified QuotePrefix and QuoteSuffix values do not match.</value>
+  </data>
+  <data name="ADP_BadParameterName" xml:space="preserve">
+    <value>Specified parameter name '{0}' is not valid.</value>
+  </data>
+  <data name="ADP_MultipleReturnValue" xml:space="preserve">
+    <value>Multiple return value parameters are not supported.</value>
+  </data>
+  <data name="ADP_EmptyString" xml:space="preserve">
+    <value>Expecting non-empty string for '{0}' parameter.</value>
+  </data>
+  <data name="ADP_EmptyArray" xml:space="preserve">
+    <value>Expecting non-empty array for '{0}' parameter.</value>
+  </data>
+  <data name="ADP_InvalidArgumentLength" xml:space="preserve">
+    <value>The length of argument '{0}' exceeds it's limit of '{1}'.</value>
+  </data>
+  <data name="ADP_MustBeReadOnly" xml:space="preserve">
+    <value>{0} must be marked as read only.</value>
+  </data>
+  <data name="ConfigProviderNotFound" xml:space="preserve">
+    <value>Unable to find the requested .Net Framework Data Provider.  It may not be installed.</value>
+  </data>
+  <data name="ConfigProviderInvalid" xml:space="preserve">
+    <value>The requested .Net Framework Data Provider's implementation does not have an Instance field of a System.Data.Common.DbProviderFactory derived type.</value>
+  </data>
+  <data name="ConfigProviderNotInstalled" xml:space="preserve">
+    <value>Failed to find or load the registered .Net Framework Data Provider.</value>
+  </data>
+  <data name="ConfigProviderMissing" xml:space="preserve">
+    <value>The missing .Net Framework Data Provider's assembly qualified name is required.</value>
+  </data>
+  <data name="AZURESQL_GenericEndpoint" xml:space="preserve">
```

Comment:
are these strings live? seem odd ones

I think I tried to remove all dead strings, then regretted it because each time I had to pull in more shared code, I needed to add the strings back. Not sure whether I just went back to all the strings.

Reply:
You are right. You removed them but I went back to all strings.
I'll add a TODO to remove unused ones now.

Reply:
For what it's worth, I removed all the dead strings using regexes and diffs, but then when I have a regex hammer everything looks like a nail. You might prefer a script.